src/assets/opplasting.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Dere burde kjøre minify på svg'ene. Jeg bruker dette verktøyet https://www.svgminify.com/.

Reply:
Ah, nice. Minifyet to av de i 49bc4d1, den siste ble ikke mindre.